### PR TITLE
send payments, reactions, typing, stickers with new auxin content merging

### PR DESCRIPTION
**Pull request by technillogue**
This depends on https://github.com/mobilecoinofficial/auxin/pull/64

- add auxin typing messages
- fix AuxinMessage.typing and drop debugs
- add sticker
- fix u8
- new auxin
- also switch over sticker and payment to new content
- add auxin reactions

### CODE REVIEW

File: forest/core.py [hunk 1]
```diff
@@ -493,55 +505,71 @@ async def send_reaction(self, target_msg: Message, emoji: str) -> None:
...
+                "dataMessage": None,
+                "typingMessage": {
+                    "action": "STOPPED" if stop else "STARTED",
+                    "timestamp": int(time.time() * 1000),
+                },
+            }
             if group:
-                content = await self.typing_message_content(stop, group)
+                content["typingMessage"]["groupId"] = group
                 await self.send_message(None, "", group=group, content=content)
```
Comment (transfaeries) [inline forest/core.py:535]: Needs to send it in the group instead of to the user in DM

File: forest/core.py [hunk 2]
```diff
@@ -472,9 +472,9 @@ async def respond(
     ) -> str:
         """Respond to a message depending on whether it's a DM or group"""
         logging.debug("responding to %s", target_msg.source)
-        if not target_msg.source:
+        if not target_msg.source or target_msg.uuid:
             logging.error(json.dumps(target_msg.blob))
-        if not utils.AUXIN and target_msg.group:
+        if target_msg.group:
             return await self.send_message(
                 None, msg, group=target_msg.group, **other_params
```
Comment (transfaeries) [inline forest/core.py:479]:
```suggestion
                None, msg, group=target_msg.group_id, **other_params
```

File: forest/core.py [hunk 3]
```diff
@@ -493,55 +505,71 @@ async def send_reaction(self, target_msg: Message, emoji: str) -> None:
...
         stop: bool = False,
         recipient: str = "",
         group: str = "",
     ) -> None:
-        "Send a typing indicator to the person or group the message is from"
         # typing indicators last 15s on their own
         # https://github.com/signalapp/Signal-Android/blob/master/app/src/main/java/org/thoughtcrime/securesms/components/TypingStatusRepository.java#L32
+        "Send a typing indicator to the person or group the message is from"
         if msg:
             group = msg.group or ""
```
Comment (transfaeries) [inline forest/core.py:523]:
```suggestion
            group = msg.group_id or ""
```

File: forest/core.py [hunk 4]
```diff
@@ -493,55 +505,71 @@ async def send_reaction(self, target_msg: Message, emoji: str) -> None:
...
+                    "size": 17540,
+                    "width": 512,
+                },
+                "packId": u8("pPYIEA9J4JkrZ2DyuXG4pw=="),
+                "packKey": u8("R2ZbM8Tz2N49WYY8yEWXPLML4JC/w1tu0naLoj5P1eY="),
+                "stickerId": 0,
+            }
+            content = {"dataMessage": {"body": None, "sticker": stick}}
+            if msg.group:
+                await self.send_message(None, "", group=msg.group, content=content)
```
Comment (transfaeries) [inline forest/core.py:567]:
```suggestion
                await self.send_message(None, "", group=msg.group_id, content=content)
```

File: sample_bots/tiamat.py [hunk 5]
```diff
@@ -949,7 +946,6 @@ async def response_monitor(self) -> None:
                 group=response.group,
```
Comment (transfaeries) [inline sample_bots/tiamat.py:946]:
```suggestion
                group=response.group_id,
```

File: sample_bots/tiamat.py [hunk 6]
```diff
@@ -766,7 +764,6 @@ async def send_sequential_messages(self) -> None:
                     group=step.message.group,
```
Comment (transfaeries) [inline sample_bots/tiamat.py:764]:
```suggestion
                    group=step.message.group_id,
```

**Review by transfaeries — CHANGES_REQUESTED** [review]
Ok, so just fix these bugs and it should be good

File: forest/core.py (hunk 1 above)
Comment (technillogue) [reply forest/core.py:535]: wait what was the issue here?

**Pull request merged**